cluster/install/install.go

```diff
@@ -4,8 +4,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
-	"github.com/karmada-io/api/cluster"
-	clusterv1alpha1 "github.com/karmada-io/api/cluster/v1alpha1"
+	"github.com/karmada-io/karmada/pkg/apis/cluster"
```

Comment:
Ah...
The main reason why we using the `api` repo is users don't need to inherit the whole dependencies at `karmada` repo.

Reply:
Thanks for your comment.